README.md

```diff
@@ -90,3 +90,8 @@ BEGIN
     RETURN c;
 END
 ```
+
+# Import & Export
+
+This is now possible to import or export data in different part of the site. Both of them can be done using CSV, XLS and ODS files.
```

Comment:
Plutôt It is now possible
Je mettrais plutot 'in the application', sans dire different part (pas la peine de préciser, sinon faudrait dire où). 
Une seule phrase Blablabla export data in the app in different formats: CSV, XLS or ODS.